scheduler/job/job.go

```diff
@@ -207,6 +209,52 @@ func (j *job) preheat(ctx context.Context, data string) error {
 	return nil
 }
 
+// preheatAllPeers preheats job by all peers, only suoported by v2 protocol.
+func (j *job) preheatAllPeers(ctx context.Context, req *internaljob.PreheatRequest) error {
+	eg, _ := errgroup.WithContext(ctx)
+	for _, host := range j.resource.HostManager().LoadAll() {
+		target := fmt.Sprintf("%s:%d", host.IP, host.Port)
+		eg.Go(func() error {
```

Comment:
Shouldn't we combine this with `batchSize` to limit the number of simultaneous tasks?